src/backend/github/branch/index.ts

```diff
@@ -1,7 +1,7 @@
-import transformBranch from "../transforms/branch";
+import transformBranch from "../../transforms/branch";
 import { Context } from "probot/lib/context";
 import issueKeyParser from "jira-issue-key-parser";
-import { isEmpty } from "../jira/util/isEmpty";
+import { isEmpty } from "../../../common/isEmpty";
```

Comment:
I like that the new structure makes each "module" have its own folder with a dedicated `index.ts` file that serves as an API into that module. 

We probably should enforce that more in the future. Currently, we are still referencing functions like `isEmpty()` from anywhere without going through the "API" of the `common` module. Although, that's probably alright for the `common` module, but not for modules with business logic in them.